formats/MT.json

```diff
@@ -1,9 +1,10 @@
 {
-  "Description": "MA : AAA 1234",
+  "Description": "MA : LLL NNNN",
   "RedundantCharacters": " -",
-  "ValidationRegex": "^[A-Z]{3}[0-9]{4}$",
+  "ValidationRegex": "^[a-zA-Z]{3}[0-9]{4}$",
```

Comment:
Would it be simpler to assume we would apply the regex only to upper case strings?